metaflow/plugins/argo/argo_workflows.py

```diff
@@ -276,6 +276,23 @@ def get_existing_deployment(cls, name):
                 )
         return None
 
+    @classmethod
+    def list(cls, name, states):
+        client = ArgoClient(namespace=KUBERNETES_NAMESPACE)
+        try:
+            workflow_template = client.get_workflow_template(name)
```

Comment:
Is this needed? what happens if I delete the workflow template but still want to list executions for the deleted template?